backend/src/main/java/planit/people/preparation/GoogleConnector/GoogleConnector.java

```diff
@@ -144,42 +141,75 @@ private void setRefreshAndExpiry() {
-                .build();
-        Userinfo userinfo = oauth2.userinfo().get().execute();
-        System.out.println("userinfo: " + userinfo);
-
-        // List the next 10 events from the primary calendar.
-        DateTime now = new DateTime(System.currentTimeMillis());
-        CalendarList calendarList = service.calendarList().list().execute();
-
-        List<CalendarListEntry> items = calendarList.getItems();
-        if (items.isEmpty()) {
-            System.out.println("No upcoming events found.");
-        } else {
-            System.out.println("Calendars: ");
-
-            for (CalendarListEntry calendar : items) {
-                System.out.printf("%s (%s)\n", calendar.getSummary(), calendar.getId());
-                Events events = service.events().list(calendar.getId()).setTimeMin(now).execute();
-                List<Event> eventList = events.getItems();
-                System.out.println("events for: "+  calendar.getSummary());
-                if (eventList.isEmpty()) {
-                    System.out.println("No upcoming events found.");
-                } else {
-                    System.out.println("Upcoming events");
-                    for (Event event : eventList) {
-                        System.out.printf("%s (%s)\n", event.getSummary(),event.getICalUID());
-                    }
-                }
-            }
+    public CalendarResponse createEvent(DTO_NewEventDetail newEventDetail, DateTime startDate) throws IOException {
+        Event event = new Event()
+                .setSummary(newEventDetail.summary())
+                .setLocation(newEventDetail.location())
+                .setDescription(newEventDetail.description());
+        EventDateTime start = new EventDateTime()
+                .setDateTime(startDate)
+                .setTimeZone(TIME_ZONE_SPECIFIER);
+        event.setStart(start);
+        DateTime endDateTime = new DateTime(startDate.getValue() + (newEventDetail.duration() * 60 * 1000));
+        EventDateTime end = new EventDateTime()
+                .setDateTime(endDateTime)
+                .setTimeZone(TIME_ZONE_SPECIFIER);
+        event.setEnd(end);
+        String[] recurrence = new String[]{"RRULE:FREQ=DAILY;COUNT=1"};
+        event.setRecurrence(Arrays.asList(recurrence));
+        List<EventAttendee> attendees = new ArrayList<>();
+        for (String attendee : newEventDetail.attendees()) {
+            attendees.add(new EventAttendee().setEmail(attendee));
+        }
+        event.setAttendees(attendees);
+
+        EventReminder[] reminderOverrides = new EventReminder[]{
+                new EventReminder().setMethod("email").setMinutes(24 * 60),
+                new EventReminder().setMethod("popup").setMinutes(10),
+        };
+        Event.Reminders reminders = new Event.Reminders()
+                .setUseDefault(false)
+                .setOverrides(Arrays.asList(reminderOverrides));
+        event.setReminders(reminders);
```

Comment:
I don't like how `event` has setters called across whole method. Call the all in the end to make it look cleaner and simpler to read